guides/common/modules/proc_installing-the-kernelcare-plug-in.adoc

```diff
@@ -0,0 +1,12 @@
+[id="Installing_the_KernelCare_Plug-in_{context}"]
+= Installing the KernelCare Plug-in
+
+Use the following procedure to install the KernelCare plug-in.
+
+.Procedure
+* Install the plug-in on your {ProjectServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --enable-foreman-plugin-kernel-care
```

Comment:
Note that today this doesn't exist yet (not even in nightly) and it won't in 3.4 or older (unless we do cherry picks). Please consider that when merging this.

Reply:
Saw this too late, so I'll revert this particular change. Please see https://github.com/theforeman/foreman-documentation/pull/1882